cdap-unit-test/src/main/java/co/cask/cdap/test/TestBase.java

```diff
@@ -384,74 +350,162 @@ private static void cleanDir(File dir) {
   }
 
   /**
+   * Deploys an {@link Application}. The {@link co.cask.cdap.api.flow.Flow Flows} and
+   * {@link co.cask.cdap.api.procedure.Procedure Procedures} defined in the application
+   * must be in the same or children package as the application.
+   *
+   * @param applicationClz The application class
+   * @return An {@link co.cask.cdap.test.ApplicationManager} to manage the deployed application.
+   */
+  protected static ApplicationManager deployApplication(Id.Namespace namespace,
+                                                        Class<? extends Application> applicationClz,
+                                                        File... bundleEmbeddedJars) {
+    ApplicationManager applicationManager = getTestManager().deployApplication(namespace, applicationClz,
+                                                                               bundleEmbeddedJars);
+    applicationManagers.add(applicationManager);
+    return applicationManager;
+  }
+
+
+  /**
+   * Deploys an {@link Application}. The {@link co.cask.cdap.api.flow.Flow Flows} and
+   * {@link co.cask.cdap.api.procedure.Procedure Procedures} defined in the application
+   * must be in the same or children package as the application.
+   *
+   * @param applicationClz The application class
+   * @return An {@link co.cask.cdap.test.ApplicationManager} to manage the deployed application.
+   */
+  protected static ApplicationManager deployApplication(Class<? extends Application> applicationClz,
+                                                        File... bundleEmbeddedJars) {
+    return deployApplication(Constants.DEFAULT_NAMESPACE_ID, applicationClz, bundleEmbeddedJars);
+  }
+
+  /**
+   * Clear the state of app fabric, by removing all deployed applications, Datasets and Streams.
+   * This method could be called between two unit tests, to make them independent.
+   */
+  protected static void clear() {
```

Comment:
Yeah, it will be an internal thing for now.. That could change when we allow tests to toggle between `UnitTestManager` and `IntegrationTestManager`, but for now it's not necessary.